### ISSUE TITLE
Double Multiplied Loss weights

### ISSUE THREAD

**Issue by WillCheung2016** [ctom2/colie#2]
For the [lines](https://github.com/ctom2/colie/blob/77fec3327bcfa48e29a9c41659f4a0874c2deaf4/colie.py#L65C9-L73C9), all loss weights to each loss term seem to be multiplied twice before summing into the total loss.

loss_spa = torch.mean(torch.abs(torch.pow(illu_lr - img_v_lr, 2))) * opt.alpha
loss_tv  = l_TV(illu_lr) * opt.beta
loss_exp = torch.mean(l_exp(illu_lr)) * opt.gamma
loss_sparsity = torch.mean(img_v_fixed_lr) * opt.delta
loss = loss_spa * opt.alpha + loss_tv * opt.beta + loss_exp * opt.gamma + loss_sparsity * opt.delta

**Comment by ctom2** [comment]
Fixed
